osidb/models.py

```diff
@@ -2919,6 +2933,76 @@ def validate(self, *args, **kwargs):
         super().clean_fields(*args, **kwargs)
 
 
+class PackageManager(ACLMixinManager, TrackingMixinManager):
+    @staticmethod
+    def create_package(flaw, package, **extra_fields):
+        """
+        Return a new Package, or update an existing Package without saving.
+        Unlike other similar manager methods in this file, new Package
+        instance is saved into database to allow relationships with PackageVer.
+        """
+        try:
+            package = Package.objects.get(flaw=flaw, package=package)
+            for attr, value in extra_fields.items():
+                setattr(package, attr, value)
+            return package
+
+        except ObjectDoesNotExist:
+            package = Package(flaw=flaw, package=package, **extra_fields)
+            package.save()
+            return package
```

Comment:
You can put `return package` at the end of the method.

Reply:
Fixed in commit [c09a6b4](https://github.com/RedHatProductSecurity/osidb/pull/307/commits/c09a6b4125ae40a2bf6f334ba66b9e0b5bc8bfac)